ext/oboe_metal/extconf.rb

```diff
@@ -56,34 +50,31 @@
 
 ao_clib = "liboboe-1.0-#{ao_arch}.so.0.0.0"
 ao_item = File.join(ao_path, ao_clib)
-ao_checksum_item = "#{ao_item}.sha256"
+ao_checksum_file = File.join(ao_lib_dir, "#{ao_clib}.sha256")
 clib = File.join(ao_lib_dir, ao_clib)
 
 retries = 3
 success = false
 while retries > 0
   begin
-    # download
-    if RUBY_VERSION < '2.5.0'
-      download = open(ao_item, 'rb')
-      checksum = open(ao_checksum_item, 'r').read.chomp
-    else
-      download = URI.open(ao_item, 'rb')
-      checksum = URI.open(ao_checksum_item, 'r').read.chomp
-    end
+    download = RUBY_VERSION < '2.5.0' ? open(ao_item, 'rb') : URI.open(ao_item, 'rb')
     IO.copy_stream(download, clib)
+
     clib_checksum = Digest::SHA256.file(clib).hexdigest
     download.close
+    checksum =  File.read(ao_checksum_file).strip
 
-    # verify_checksum
+    # unfortunately these messages only show if the install command is run
+    # with the `--verbose` flag
     if clib_checksum != checksum
       $stderr.puts '== ERROR ================================================================='
-      $stderr.puts 'Checksum Verification failed for the c-extension of the appoptics_apm gem.'
-      $stderr.puts 'appoptics_apm will not instrument the code. No tracing will occur.'
-      $stderr.puts 'Contact support@appoptics.com if the problem persists.'
+      $stderr.puts 'Checksum Verification failed for the c-extension of the appoptics_apm gem'
+      $stderr.puts 'Installation can not continue'
+      $stderr.puts "\nChecksum cut into gem:        #{checksum}"
+      $stderr.puts "Checksum caluclated from lib: #{clib_checksum}"
```

Comment:
Minor, typo for `calculated`.  And not sure if "checksum cut into gem" is standard lingo, may be clearer to have something like `checksum packaged with the gem, versus checksum calculated from download`

Reply:
oops!